### PR TITLE
chore: update dependencies

### PR DESCRIPTION
**Pull request by dzhavat**
### Workspace
- update dependencies
- update NOTICE file
- change import variable name

### DataHub
- use official `@azure/msal-angular` instead of custom code

### Energy Origin
- bump project version

## References

- #563

### CODE REVIEW

File: tsconfig.base.json [hunk 1]
```diff
@@ -228,7 +228,8 @@
       "@energinet-datahub/gf/util-browser": [
         "libs/gf/util-browser/src/index.ts"
       ],
-      "@energinet-datahub/watt": ["libs/ui-watt/src/index.ts"]
+      "@energinet-datahub/watt": ["libs/ui-watt/src/index.ts"],
+      "rxjs": ["./node_modules/rxjs"]
```
Comment (frontend-specialisten) [inline tsconfig.base.json:232]: Question: why is this necessary?

Comment (frontend-specialisten) [reply tsconfig.base.json:232]: RxJSv6 is the default version of RxJS in MSAL-Angular v2. In order to use the installed version of RxJSv7 for this sample, "rxjs": ["./node_modules/rxjs"] must be added to the compilerOptions object's paths object in tsconfig.json. This tells the compiler to use the version of RxJS specified in package.json (RxJSv7).

Comment (dzhavat) [reply tsconfig.base.json:232]: Exactly. I hope it works :)

**Review by frontend-specialisten — DISMISSED** [review]

**Review by frontend-specialisten — APPROVED** [review]
